Mage.Sets/src/mage/cards/t/TimeToFeed.java

```diff
@@ -41,19 +29,20 @@ public final class TimeToFeed extends CardImpl {
     public TimeToFeed(UUID ownerId, CardSetInfo setInfo) {
         super(ownerId,setInfo,new CardType[]{CardType.SORCERY},"{2}{G}");
 
-
         // Choose target creature an opponent controls. When that creature dies this turn, you gain 3 life. 
-        this.getSpellAbility().addEffect(new TimeToFeedTextEffect());
+        this.getSpellAbility().addEffect(new CreateDelayedTriggeredAbilityEffect(
+                new WhenTargetDiesDelayedTriggeredAbility(new GainLifeEffect(3)),
+                true,
+                "Choose target creature an opponent controls. "
+        ));
         // Target creature you control fights that creature.
         Effect effect = new FightTargetsEffect();
         effect.setText("Target creature you control fights that creature. " +
                 "<i>(Each deals damage equal to its power to the other.)</i>");
         this.getSpellAbility().addEffect(effect);
 
-        Target target = new TargetCreaturePermanent(StaticFilters.FILTER_OPPONENTS_PERMANENT_CREATURE);
-        this.getSpellAbility().addTarget(target);
+        this.getSpellAbility().addTarget(new TargetPermanent(StaticFilters.FILTER_OPPONENTS_PERMANENT_CREATURE));
```

Comment:
You should probably put a not here that the ordering of these two targets is important. WhenTargetDiesDelayedTriggeredAbility grabs the first target, but there's no indication of that here.

Reply:
Effects in xmage default to using the first target of an ability (with an unfortunate exception in `DamageTargetEffect`, which defaults to using *all* the targets and requires an explicit constructor argument to make it respect the `TargetPointer` system)

Actually, there's a bit of an issue here in that the way `CreateDelayedTriggeredAbilityEffect` is implemented *only* allows `WhenTargetDiesDelayedTriggeredAbility` to work with the first target, but fixing that would require a significant refactor to a very widely-used class and luckily no cards currently need it.